codecov_auth/commands/owner/interactors/set_yaml_on_owner.py

```diff
@@ -49,10 +50,12 @@ def convert_yaml_to_dict(self, yaml_input):
             raise ValidationError(message)
```

Comment:
I'd guess in a perfect world we would want `validate_yaml` (in [shared](https://github.com/codecov/shared/blob/fffdcf08be320e5feb14111d2cea4e5e8bd54ad0/shared/yaml/validation.py#L16)) to reject if any reserved keys (like `to_string`) are in it. It's probably a lot of work for something unlikely, though so can let it be

Reply:
Oh never mind I had missed the note in the PR description! Sounds like that will get covered later - nice!